graphql/student/mutations.ts

```diff
@@ -109,6 +113,7 @@ export async function updateStudent(context: GraphQLContext, student: Student, u
             registrationSource: ensureNoNull(registrationSource),
             state: ensureNoNull(state),
             aboutMe: ensureNoNull(aboutMe),
+            lastTimeCheckedNotifications: ensureNoNull(lastTimeCheckedNotifications),
```

Comment:
is this correct? field can be null

Reply:
is this the right mutation? we are usually updating user fields and last checked date separately, right?

Reply:
same question with other roles

Reply:
ensureNoNull replaces null with undefined (which is desired, in Prisma undefined means 'ignore' whether null means 'set to NULL'), so the use here is correct. 

meUpdate is supposed to update the user record with simple write operations, so as long as the time is only written I think it's fine, if you plan to add more logic here then I guess it would deserve a separate mutation.